### PR TITLE
Don 649 accesibility

### PR DESCRIPTION
**Pull request by bdsl**
Use appropriate heading levels for cards and footer, resolving issue reported by Ligghthouse in chrome dev tools.

image

### CODE REVIEW

File: src/app/home/home.component.html [hunk 1]
```diff
@@ -12,8 +12,9 @@
 
     <biggive-grid
       column-count="3">
-      <biggive-basic-card *ngIf="greenMatchApplicationsOpen"
+      <biggive-basic-card
         space-below='4'
+        heading-level = 2
```
Comment (alihejazi) [inline src/app/home/home.component.html:17]: For consistency:
```suggestion
        heading-level=2
```

Comment (alihejazi) [reply src/app/home/home.component.html:17]: @bdsl Also not sure but pls test locally that it works when passed as an int and not as a string.

Comment (bdsl) [reply src/app/home/home.component.html:17]: @alihejazi I have tested it locally, it works. The screenshot in the PR descriptions shows the issue goes away in Lighthouse.

File: src/app/home/home.component.html [hunk 2]
```diff
@@ -23,21 +24,11 @@
...
-        button-label='Read more'
-        button-colour-scheme='clear-primary'
-        class="basic-card-padding">
-      </biggive-basic-card>
 
-      <biggive-basic-card *ngIf="championsForChildrenApplicationsOpen"
+      <biggive-basic-card
         space-below='4'
         icon-colour='brand-5'
+        heading-level = 2
```
Comment (alihejazi) [inline src/app/home/home.component.html:31]:
```suggestion
        heading-level=2
```

File: src/app/home/home.component.html [hunk 3]
```diff
@@ -46,20 +37,10 @@
...
-        subtitle='Applications open 9 Jan 2023'
-        button-url='https://blog.thebiggive.org.uk/champions-for-children/'
-        button-label='Read more'
-        button-colour-scheme='clear-primary'
-        class="basic-card-padding">
-      </biggive-basic-card>
       <biggive-basic-card
         space-below='4'
         icon-colour='primary'
+        heading-level = '2'
```
Comment (alihejazi) [inline src/app/home/home.component.html:43]:
```suggestion
        heading-level='2'
```

**Comment by bdsl** [comment]
I think in principle we could merge this as-is without breaking anything, but maybe it's worth waiting to include the new version and avoid referencing a prop that doesn't exist in the current version.

**Review by alihejazi — APPROVED** [review]

**Pull request merged**